lighthouse-core/audits/seo/font-size.js

```diff
@@ -285,16 +289,16 @@ class FontSize extends Audit {
 
     let explanation;
     if (!passed) {
-      const percentageOfFailingText = parseFloat((100 - percentageOfPassingText).toFixed(2));
+      const percentageOfFailingText = parseFloat((100 - percentageOfPassingText).toFixed(2)) / 100;
       let disclaimer = '';
 
       // if we were unable to visit all text nodes we should disclose that information
       if (visitedTextLength < totalTextLength) {
         const percentageOfVisitedText = visitedTextLength / totalTextLength * 100;
         disclaimer = ` (based on ${percentageOfVisitedText.toFixed()}% sample)`;
       }
-
-      explanation = `${percentageOfFailingText}% of text is too small${disclaimer}.`;
+      explanation = str_(UIStrings.explanation,
+        {decimalProportion: percentageOfFailingText, disclaimer});
```

Comment:
So this (combining) would require a few additions elsewhere in the i18n handling right? (this is where I admit I don't know how i18n works here yet)

```
if (visitedTextLength < totalTextLength) {
	const percentageOfVisitedText = visitedTextLength / totalTextLength * 100;
	disclaimer = str_(UIStrings.explanation, {percentage: percentageOfVisitedText.toFixed()})
}
explanation = str_(UIStrings.explanation,
    {decimalProportion: percentageOfFailingText} + disclaimer)
```

That, or have two separate UIStrings for cases w/ the disclaimer and w/o.

Reply:
I think we'll have to do `str_(UIStrings.explanation, {decimalProportion) + ' ' + str_(UIStrings.disclaimer, {percentage})`

it's not super kosher but it's better than having a random english string

the first argument to `str_` must always be a statically known value

Reply:
The i18n replacement in the end can't replace 2 lookup strings can it, the output ends up like this afaik:
`"lighthouse-core/audits/seo/font-size.js | explanation # 1 lighthouse-core/audits/seo/font-size.js | disclaimer # 0."`
b/c it can't lookup 2 replacements in 1 string?  Should we add that, or is there an option I'm not setting/seeing?

Reply:
Oh right we defer them to the end now, this is gonna be tricky 🤔 we might want to come up with a new format to handle this type of thing, like an `I18NCombinedString` object that gets replaced instead of just a normal string

Reply:
+1 to some concat string handling (*not* nesting :).

As long as the constituent strings are findable statically, the human translators will understand and replacement should work fine after concatenating. We just need to get i18n.js to handle it without adding too much complexity.

Or what's the combinatorics on this audit? We could just defer on this and include multiple (partially redundant) explanation UIStrings.

Reply:
> Or what's the combinatorics on this audit?

It's just two. I vote for deferring.

Reply:
Deferred by splitting into explanation with disclaimer and without  

_punted_ :football: